imobil.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Imobil extends Model
```

Comment:
The file name should be `Imobil` as your class name

Reply:
also `imobil` is a Romanian word, please try to find an English one. I propose `Property`, these changes imply renaming your table

Reply:
change into Imobil.php